pkg/kms/webkms/remotekms.go

```diff
@@ -41,64 +43,46 @@ const (
 	// LocationHeader is remoteKMS http header set by the key server (usually to identify a keystore or key url).
 	LocationHeader = "Location"
 
-	// KeyBytesHeader is remoteKMS http header set by the key server (to contain an base64URL encoded public key).
-	KeyBytesHeader = "KeyBytes"
+	// SecretHeader is the keystore secret (share) in the key server.
+	SecretHeader = "Secret"
+
+	// InvokerHeader is remoteKMS http request invoker.
+	InvokerHeader = "Invoker"
```

Comment:
This header is not needed. The invoker is picked by the service from the `keyId` in `Signature` header.

Reply:
The invoker is always the sender of the HTTP message, who authenticates the message with a signature computed using the `keyID`.

Reply:
I am trying to understand why at all we should specify headers here. Shouldn't this be done by the one who configures the framework instance?

Reply:
yes, I know, using the optional function for adding headers (that will invoke the authz key server for getting the cap key) will call the the auth service with keyID and set the resulting `Signature` header

I added it because when creating a keystore, we explicitly set the `controller` in the call so we can store the controller in the local remoteKMS config storage for future calls where the controller is used as the invoker (for preventive measures in addition to the keyID signatures).

Reply:
> (for preventive measures in addition to the keyID signatures).

The server's zcap-ld authz will not use this `invoker` header to determine the invoker of the capability. Instead, the `keyID` parameter of the `Signature` header will be used.

Reply:
I will remove controller header

Reply:
Updated. Now `secret` and `controller` headers are not managed by remoteKMS anymore (instead a user must provider the optional headers function that should set any auhorization info needed in the request)